lib/graphql/schema.flow.js

```diff
@@ -333,11 +333,22 @@ export type CategorizationType = $Values<typeof CategorizationTypeValues>;
 
 export type CategorizeTransactionForDeclarationResponse = {|
   __typename?: 'CategorizeTransactionForDeclarationResponse',
+  categoryCode?: ?$ElementType<Scalars, 'String'>,
   elsterCode?: ?$ElementType<Scalars, 'String'>,
```

Comment:
The file is automatically generated. Because the backend supports backward compatibility, therefore we still have `elsterCode` in request/response.